web/src/components/EpisodeViewer.vue

```diff
@@ -171,6 +174,15 @@ const load = async () => {
         player.value.on('ended', () => {
             emit('next')
         })
+        player.value.on('play', () => {
+            analytics.track('video_played', {
+                videoId: props.episode.id,
+                referenceId: referenceId,
+                data: {
+                    // Whatever data we want to send
```

Comment:
For QC maybe it would be smart to add `ageGroup` right away?
Then we could join the tables and compare the 2 felds to make sure we are matching things as expected.
I'm in addition to that RS Session ID should also match.

Reply:
Added it to all `analytics.track` calls 👍